src-tauri/src/main.rs

```diff
@@ -211,4 +215,12 @@ fn get_main_window(handle: &AppHandle) -> Window {
         let main = handle.get_window("AD4MIN");                
         main.expect("Couldn't get main window right after creating it")
     }
+}
+
+fn log_error(window: &Window) {
+    dialog::message(
+        Some(window), 
+        "Error", 
+        "Something went wrong while starting ad4m-executor please check the logs"
```

Comment:
I think this should read: "There was an error with AD4MIN. Restarting may fix this, otherwise please contact the AD4M team for support."